source/renderer/app/components/wallet/transactions/WalletTransactions.js

```diff
@@ -0,0 +1,183 @@
+    this.props.onFilter(filterOptions);
+    this.onFilterDialogClose();
+  };
+
+  render() {
+    const { intl } = this.context;
+    const { isFilterDialogOpen, isScrolling } = this.state;
+    const {
+      activeWallet,
+      transactions,
+      filterOptions,
+      deletePendingTransaction,
+      onLoadMore,
+      hasMoreToLoad,
+      isLoadingTransactions,
+      onOpenExternalLink,
+      getUrlByType,
+      isDeletingTransaction,
+      totalAvailable,
+      currentDateFormat,
+      currentTimeFormat,
+      currentNumberFormat,
+      currentLocale,
+      onRequestCSVFile,
+      defaultFilterOptions,
+      populatedFilterOptions,
+    } = this.props;
+
+    // Guard against potential null values
+    // @TX TODO - display NoWallets?
+    if (!filterOptions || !activeWallet) return null;
+
+    let walletTransactions = null;
+    // const { searchLimit } = filterOptions;
+    const numberOfFilterDimensionsApplied = getNumberOfFilterDimensionsApplied(
+      filterOptions
+    );
+    const noTransactionsLabel = intl.formatMessage(messages.noTransactions);
+
+    const isRestoreActive = activeWallet && activeWallet.isRestoring;
+
+    const isFilterDisabled =
+      !transactions.length && !numberOfFilterDimensionsApplied;
+
+    if (!transactions.length) {
+      walletTransactions = numberOfFilterDimensionsApplied ? (
+        <FilterResultInfo filtered={0} total={totalAvailable} />
+      ) : (
+        <WalletNoTransactions label={noTransactionsLabel} />
+      );
+    } else {
+      walletTransactions = (
+        <WalletTransactionsList
+          transactions={transactions}
+          deletePendingTransaction={deletePendingTransaction}
+          isLoadingTransactions={isLoadingTransactions}
+          isRestoreActive={isRestoreActive}
+          onLoadMore={onLoadMore}
+          hasMoreToLoad={hasMoreToLoad}
+          walletId={activeWallet.id}
+          isDeletingTransaction={isDeletingTransaction}
+          formattedWalletAmount={formattedWalletAmount}
+          onOpenExternalLink={onOpenExternalLink}
+          getUrlByType={getUrlByType}
+          currentLocale={currentLocale}
+          currentTimeFormat={currentTimeFormat}
+          currentDateFormat={currentDateFormat}
+          isRenderingAsVirtualList
+        />
+      );
+    }
+
+    return (
+      <WalletTransactionsListScrollContext.Provider
+        value={{ setIsScrolling: this.setIsScrolling }}
+      >
+        <div className={styles.component}>
+          <WalletTransactionsHeader
+            numberOfFilterDimensionsApplied={numberOfFilterDimensionsApplied}
+            numberOfTransactions={transactions.length}
+            onRequestCSVFile={onRequestCSVFile}
+            onFilterDialogOpen={this.onFilterDialogOpen}
+            isScrolling={isScrolling}
+            isFilterDisabled={isFilterDisabled}
+          >
+            {isFilterDialogOpen && (
+              <FilterDialog
+                locale={currentLocale}
```

Comment:
@daniloprates are we using this property in the FilterDialog component?

Reply:
Yes, it uses for specific date input formattings

Reply:
Cool